cmd/meroxa/turbine_cli/javascript/deploy.go

```diff
@@ -38,19 +38,14 @@ func NeedsToBuild(path string) (bool, error) {
 	return strconv.ParseBool(match[1])
 }
 
-func BuildApp(path string) (string, error) {
+func CreateDockerfile(ctx context.Context, l log.Logger, path string) error {
```

Comment:
Updated this function that it just calls out to turbine-js to add the Dockerfile to the project root, similar to turbine-go